### PR TITLE
b2bevents description updates

### PR DESCRIPTION
**Pull request by prabhum2**
Please link to the issue #…

### CODE REVIEW

File: components/mixins/experience-event/events/scorechanged.schema.json [hunk 1]
```diff
@@ -29,52 +29,52 @@
...
                   "title": "Score Atribute ID",
                   "type": "string",
-                  "description": "Unique identifier for score attribute."
+                  "description": "Unique ID of the lead score attribute."
                 },
                 "xdm:scoreAttributeName": {
                   "title": "Score Atribute Name",
                   "type": "string",
-                  "description": "friendly name for the score."
+                  "description": "Friendly name of the lead score attribute."
```
Comment (peichiang) [inline components/mixins/experience-event/events/scorechanged.schema.json:37]: Friendly name of the lead score attribute. => Name of the lead score attribute.

File: components/mixins/experience-event/events/statusincampaignprogressionchanged.schema.json [hunk 2]
```diff
@@ -29,42 +29,42 @@
...
                   "title": "New Status ID",
                   "type": "string",
-                  "description": "Unique identifier for new status."
+                  "description": "ID of the lead's new status in the campaign."
                 },
                 "xdm:newStatusName": {
                   "title": "New Status Name",
                   "type": "string",
-                  "description": "Friendly name for new status."
+                  "description": "Friendly name of the lead's new status in the campaign."
```
Comment (peichiang) [inline components/mixins/experience-event/events/statusincampaignprogressionchanged.schema.json:42]: Friendly name of the lead’s new status in the campaign. => Name of the lead’s new status in the campaign.

File: components/mixins/experience-event/events/visit-webpage.schema.json [hunk 3]
```diff
@@ -45,12 +45,12 @@
                 "xdm:queryParamaters": {
```
Comment (peichiang) [inline components/mixins/experience-event/events/visit-webpage.schema.json:45]: xdm:queryParamaters =>xdm:queryParameters

File: components/mixins/experience-event/events/visit-webpage.schema.json [hunk 4]
```diff
@@ -45,12 +45,12 @@
...
                   "title": "Query Paramaters",
```
Comment (peichiang) [inline components/mixins/experience-event/events/visit-webpage.schema.json:46]: “title”: “Query Paramaters” => “title”: “Query Parameters”

File: components/mixins/experience-event/events/visit-webpage.schema.json [hunk 5]
```diff
@@ -45,12 +45,12 @@
...
                   "type": "string",
-                  "description": "Query Paramters used for the web page."
+                  "description": "Query Paramters used for the web page. This is normally the value of a string after ? in the url."
```
Comment (peichiang) [inline components/mixins/experience-event/events/visit-webpage.schema.json:48]: Query Paramters used for the web?page. This is normally the value of a string after `?` in the url. => Query Parameters used for the webpage. This is normally the value of a string after `?` in the url.

**Review by peichiang — CHANGES_REQUESTED** [review]

File: components/mixins/experience-event/events/emailbounced.schema.json [hunk 6]
```diff
@@ -25,42 +25,37 @@
...
               "title": "Mailing ID",
               "type": "string",
-              "description": "Unique identifier of the mailing."
+              "description": "Unique ID of the email asset."
             },
             "xdm:mailingName": {
               "title": "Mailing Name",
               "type": "string",
-              "description": "Friendly name for the mailing."
+              "description": "Friendly name of the email asset."
```
[the remark this answers is not in the document]
Comment (prabhum2) [reply components/mixins/experience-event/events/emailbounced.schema.json:33]: we typically use friendly in any non standard name description

File: components/mixins/experience-event/events/emailbounced.schema.json [hunk 7]
```diff
@@ -25,42 +25,37 @@
...
               "title": "Test Variant ID",
               "type": "string",
-              "description": "Unique identifier for the test variant."
+              "description": "Unique ID of the test variant of the email asset."
             },
             "xdm:testVariantName": {
               "title": "Test Variant Name",
               "type": "string",
-              "description": "Friendly name for the test variant."
+              "description": "Friendly name of the test variant of the email asset ."
```
[the remark this answers is not in the document]
Comment (prabhum2) [reply components/mixins/experience-event/events/emailbounced.schema.json:58]: same as above

File: components/mixins/experience-event/events/emailbouncedsoft.schema.json [hunk 8]
```diff
@@ -25,42 +25,37 @@
...
               "title": "Mailing ID",
               "type": "string",
-              "description": "Unique identifier of the mailing."
+              "description": "Unique ID of the email asset."
             },
             "xdm:mailingName": {
               "title": "Mailing Name",
               "type": "string",
-              "description": "Friendly name for the mailing."
+              "description": "Friendly name of the email asset."
```
[the remark this answers is not in the document]
Comment (prabhum2) [reply components/mixins/experience-event/events/emailbouncedsoft.schema.json:33]: same as above

**Review by mjasrotia — APPROVED** [review]

**Review by peichiang — COMMENTED** [review]
1. Many of the “Friendly name” usage instead of "Name" due to convention. Waiting for Liz to confirm.
2. We used “webpage” in most places and “web page” in some places listed below. Should we be consistent?

**Review by peichiang — APPROVED** [review]

**Pull request merged**